### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by newhinton** [nextcloud/unsplash#95] [Ideas]
If you have suggestions or improvements that you would like to see, please discuss then here!

Please refrain from asking for imageprovider here. There is a seperate discussion for that.

**Comment by cvick** [comment]
Hello! Would it be possible to have a link to the Unsplash page for the current background image at the bottom of the login / dashboard page? It could be an option that can be turned on or off in the apps settings. Sometimes I want to use an image that gets loaded as my wallpaper, so right now I open dev tools before loading my Nextcloud instance to get the Unsplash URL after the page loads.

Thanks for your time and for the app!

**Reply by newhinton** [reply]
Your suggestion has two components to it:

1. Providing a Download-Link to the current image
2. Display the Source of the image

Those two are not nessessarily the same, and in the case for unsplash, they arent.

We had a PR for the latter a while ago, but it wasnt merged because of multiple reasons. I do plan to eventually add this as a feature. It will be a small icon somewhere, which shows a popover with the author and a link to the source. (Not nessessarily the image)

I will not implement the first one. I dont think that this is something the majority of users need, especially not regularly enough, so it would be a distraction. Also those links ususally look ugly, though it could be hidden behind an icon. If enough users request this, i will rethink implementing it though.

**Comment by tigernero79** [comment]
expected an update for nextcloud 25?

**Reply by newhinton** [reply]
Support for Nextcloud 25 (Hub 3) is beeing tracked in #105. It will take a bit, but i already did a lot of the required work.

**Comment by highjumper83** [comment]
Hello, the new feature that I can choose the source of the pictures is great. It would be perfekt then I can also choose a folder in my nexcloud, there I can store pictures by myself.

**Comment by AxIauk** [comment]
Users in different regions and different network environments may have large delays in downloading images, I can display completed images in about 3-5 seconds per refresh. I would like to cache them in nextcloud in advance.

**Comment by Siggi0904** [comment]
Please push the latest version for nc 30 to the official app store.
https://apps.nextcloud.com/apps/unsplash

Thanks.

**Comment by TheBest-Name** [comment]
Can you get the app to support NC 33?
